control-plane/agents/src/bin/core/volume/operations_helper.rs

```diff
@@ -527,6 +527,55 @@ impl OperationGuardArc<VolumeSpec> {
         Ok(created_replicas)
     }
 
+    pub(super) async fn resize_volume_replicas(
+        &self,
+        registry: &Registry,
+        replicas: &Vec<Replica>,
+        requested_size: u64,
+        spec_clone: &VolumeSpec,
+    ) -> Result<(), SvcError> {
+        for replica in replicas {
+            let repl_rsc = match registry.specs().replica_rsc(&replica.uuid) {
+                Some(rsc) => rsc,
+                None => {
+                    return self
+                        .validate_update_step(
```

Comment:
don't need the validates here, see my previous review

Reply:
> don't need the validates here, see my previous review

Fixed. Will update diff on PR soon.